python/dolfinx/fem/__init__.py

```diff
@@ -57,6 +66,35 @@ def create_sparsity_pattern(a: Form):
     return _create_sparsity_pattern(a._cpp_object)
 
 
+def create_nonmatching_meshes_interpolation_data(
+    mesh_to: typing.Union[_Mesh, _Geometry_float64, _Geometry_float32],
+    element: typing.Union[_FiniteElement_float32, _FiniteElement_float64],
+    mesh_from: _Mesh,
+    cells: typing.Optional[npt.NDArray[np.int32]] = None,
+    padding: float = 1e-14,
+) -> tuple[list[np.int32], list[np.int32], list[np.floating], list[np.int32]]:
```

Comment:
It would be nice to document what each of these lists contains. @jorgensd Any ideas?

Reply:
Perhaps used a `namedtuple`?